artiq/firmware/runtime/rtio_mgt.rs

```diff
@@ -349,23 +353,36 @@ fn async_error_thread(io: Io) {
             io.until(|| csr::rtio_core::async_error_read() != 0).unwrap();
             let errors = csr::rtio_core::async_error_read();
             if errors & ASYNC_ERROR_COLLISION != 0 {
-                error!("RTIO collision involving channel {}",
-                       csr::rtio_core::collision_channel_read());
+                let channel = csr::rtio_core::collision_channel_read();
+                error!("RTIO collision error involving channel {} ({})",
+                       channel, config_channel_name(channel));
+                
             }
             if errors & ASYNC_ERROR_BUSY != 0 {
-                error!("RTIO busy error involving channel {}",
-                       csr::rtio_core::busy_channel_read());
+                let channel = csr::rtio_core::busy_channel_read();
+                error!("RTIO busy error involving channel {} ({})",
+                       channel, config_channel_name(channel));
             }
             if errors & ASYNC_ERROR_SEQUENCE_ERROR != 0 {
-                error!("RTIO sequence error involving channel {}",
-                       csr::rtio_core::sequence_error_channel_read());
+                let channel = csr::rtio_core::sequence_error_channel_read();
+                error!("RTIO sequence error involving channel {} ({})",
+                       channel, config_channel_name(channel));
             }
             SEEN_ASYNC_ERRORS = errors;
             csr::rtio_core::async_error_write(errors);
         }
     }
 }
 
+fn config_channel_name(ch_number: u16) -> String {
+    let ch_name = config::read_str(&format!("channel {}", ch_number), |r| match r {
```

Comment:
You failed to address all my points in https://github.com/m-labs/artiq/pull/1935#discussion_r917693862

Reply:
If you are taking about the config key, why is it about "name"?
Shouldn't the arg be the channel number only?

Reply:
There are many other things in config and some are potentially about other aspects of channels.

Reply:
ohh i see